.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Expected Behavior
+
+
+## Actual Behavior
+
+
+## Steps to Reproduce the Problem
+
+  1.
+  1.
```

Comment:
Steps to reproduce problem, shouldn't it be "1. 2. 3."and not "1. 1. 1." ?

Reply:
@dominikkawka this should be reflected as 1. 2. 3. if you click the `view` button you will see what I mean